airbyte_cdk/sources/file_based/config/permissions.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+import uuid
+from datetime import datetime
+from enum import Enum
+
+from pydantic.v1 import BaseModel
+
+
+class RemoteFileIdentityType(Enum):
+    USER = "user"
+    GROUP = "group"
+
+
+class RemoteFileIdentity(BaseModel):
```

Comment:
A one-senctence docstring would be very useful, explaining why this exists and why most fields are nulllable, and in what situations we want to use them